ios/pillarwallet.xcodeproj/project.pbxproj

```diff
@@ -1807,14 +1807,15 @@
 					"-ObjC",
 					"-lc++",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.pillarproject.wallet;
+				PRODUCT_BUNDLE_IDENTIFIER = "com.pillarproject.wallet-pedro";
```

Comment:
I'd say we should keep the previous value on these.

Reply:
Yeah, totally. That was just for me to run it on my iPhone